src/components/reusable/notification/notification.ts

```diff
@@ -60,10 +59,17 @@ export class Notification extends LitElement {
     error: 'failed',
   };
 
+  /** Set notification mark read prop.*/
+  @property({ type: Boolean, reflect: true })
+  markRead: any = false;
```

Comment:
Right now, "unread" is the default state, is that right? Might want to confirm that with Marco, because read/unread functionality might not be used in every case. In the case there is no unread/read functionality, I would expect "read" styles to be the default.

So what I propose is naming this prop "unread", and default = false.

Reply:
yes **unread** is default state. I am changing this prop name as **unread**

Reply:
done